feature/github-repo-importer/pkg/github/github.go

```diff
@@ -92,6 +103,68 @@ func ImportRepo(repoName string) (*Repository, error) {
 		fmt.Printf("failed to write pages.json: %v\n", err)
 	}
 
+	// =========================================================================
+	// FEATURE: GitHub Environments
+	// =========================================================================
+	// Feature flag: feature_github_environment (default: DISABLED - opt-in feature)
+	// Set feature_github_environment: true in import-config.yaml to enable environment import.
+	//
+	// When enabled, environments are imported from GitHub and managed by Terraform.
+	var allEnvironments []*github.Environment
+
+	// Check if feature is enabled (defaults to false - opt-in feature)
+	enableEnvironments := false
+	if cfg != nil && cfg.Features != nil {
+		if enabled, exists := cfg.Features[FeatureGithubEnvironment]; exists {
+			enableEnvironments = enabled
+		}
+	}
+
+	if enableEnvironments {
```

Comment:
the three ifs can be reduced. you added a function that is not used cfg.IsFeatureEnabled that you can use to reduce the number of ifs. once that is done, you won't need `enableEnvironments` any more